ui/component/claimDescription/index.js

```diff
@@ -6,11 +6,12 @@ import { getClaimMetadata } from 'util/claim';
 const select = (state, props) => {
   const { uri, description: descriptionProp } = props;
 
-  const claim = !descriptionProp && selectClaimForUri(state, uri);
+  const hasPassedDescription = 'description' in props;
+  const claim = !hasPassedDescription && selectClaimForUri(state, uri);
   const metadata = claim && getClaimMetadata(claim);
 
   return {
-    description: descriptionProp || (metadata && metadata.description),
+    description: hasPassedDescription ? descriptionProp : metadata && metadata.description,
   };
 };
```

Comment:
That was the idea at least on collections since the edited collection can return `''` as empty and the condition considers it the same as false/undefined/null, so use whatever is being passed